### PR TITLE
rnndb: de-UNK a few pfifo regs

### PR DESCRIPTION
**Pull request by kfractal**
g80 and gf100 pfifo
i'm working on something more comprehensive.  but for now i just would like to get my first ever change in :)

### CODE REVIEW

**Comment by kfractal** [comment]
fixing and pushing up the result later. closing this one.

**Comment by koriakin** [comment]
Say, where did you get that information from?

**Comment by koriakin** [comment]
Oh wait... nevermind

File: rnndb/fifo/gf100_pfifo.xml [hunk 1]
```diff
@@ -217,7 +217,14 @@
 		<bitfield pos="8" name="ACTIVE"/>
 	</reg32>
 	<array offset="0x800" name="VM_FAULT" length="32" stride="0x10">
-		<reg32 offset="0" name="CHAN" type="g80_channel"/>
+		<reg32 offset="0" name="CHAN" type="g80_channel">
```
Comment (koriakin) [inline rnndb/fifo/gf100_pfifo.xml]: Don't list bitfields in this register - setting type="g80_channel" already does that.

**Comment by mupuf** [comment]
Sweet!

File: rnndb/fifo/gf100_pfifo.xml (hunk 1 above)
Comment (kfractal) [inline rnndb/fifo/gf100_pfifo.xml]: eep.  i see that now.  thanks...

**Pull request closed without merging**